distribution/bin/seata-server.bat

```diff
@@ -98,7 +93,7 @@ if "%SKYWALKING_ENABLE%"=="true" (
   echo "apm-skywalking not enabled"
 )
 
-%JAVACMD% %JAVA_OPTS% %SKYWALKING_OPTS% -server -Xmx2048m -Xms2048m -Xmn1024m -Xss512k -XX:SurvivorRatio=10 -XX:MetaspaceSize=128m -XX:MaxMetaspaceSize=256m -XX:MaxDirectMemorySize=1024m -XX:-OmitStackTraceInFastThrow -XX:-UseAdaptiveSizePolicy -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath="%BASEDIR%"/logs/java_heapdump.hprof -XX:+DisableExplicitGC -XX:+CMSParallelRemarkEnabled -XX:+UseCMSInitiatingOccupancyOnly -XX:CMSInitiatingOccupancyFraction=75 -Xloggc:"%BASEDIR%"/logs/seata_gc.log -verbose:gc -Dio.netty.leakDetectionLevel=advanced -classpath %CLASSPATH% -Dapp.name="seata-server" -Dapp.repo="%REPO%" -Dapp.home="%BASEDIR%" -Dbasedir="%BASEDIR%" -Dspring.config.location="%BASEDIR%"/conf/application.yml -Dlogging.config="%BASEDIR%"/conf/logback-spring.xml -jar "%BASEDIR%"/target/seata-server.jar %CMD_LINE_ARGS%
+%JAVACMD% %JAVA_OPTS% %SKYWALKING_OPTS% -server -Dloader.path=../lib -Xmx2048m -Xms2048m -Xmn1024m -Xss512k -XX:SurvivorRatio=10 -XX:MetaspaceSize=128m -XX:MaxMetaspaceSize=256m -XX:MaxDirectMemorySize=1024m -XX:-OmitStackTraceInFastThrow -XX:-UseAdaptiveSizePolicy -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath="%BASEDIR%"/logs/java_heapdump.hprof -XX:+DisableExplicitGC -XX:+CMSParallelRemarkEnabled -XX:+UseCMSInitiatingOccupancyOnly -XX:CMSInitiatingOccupancyFraction=75 -Xloggc:"%BASEDIR%"/logs/seata_gc.log -verbose:gc -Dio.netty.leakDetectionLevel=advanced -classpath %CLASSPATH% -Dapp.name="seata-server" -Dapp.repo="%REPO%" -Dapp.home="%BASEDIR%" -Dbasedir="%BASEDIR%" -Dspring.config.location="%BASEDIR%"/conf/application.yml -Dlogging.config="%BASEDIR%"/conf/logback-spring.xml -jar "%BASEDIR%"/target/seata-server.jar %CMD_LINE_ARGS%
```

Comment:
how ext should be handled

Reply:
`java.ext.dirs`  has default value, so it will  overwrites the original value when you set it.
![image](https://user-images.githubusercontent.com/8758457/174484868-61076c64-fa80-4b55-9c4d-aeda57985542.png)

`loader.path` is just use for springboot loader and the default  value is null.

![image](https://user-images.githubusercontent.com/8758457/174485050-b31071e0-6c20-432b-be17-2c2c044dcdf3.png)
![image](https://user-images.githubusercontent.com/8758457/174485093-fd7e32be-03ac-4147-84a6-c15b15553ae8.png)

Reply:
I mean ext refers to the directory of skywalk plugins @slievrly

Reply:
![image](https://user-images.githubusercontent.com/8758457/174754718-0ed65741-784d-40ca-a869-f595c69b0237.png)